Dockerfile

```diff
@@ -109,7 +109,6 @@ RUN if [ ${RAILS_ENV} = "production" ]; then \
 
 EXPOSE 3000
 
-ENTRYPOINT [ "bin/docker-entrypoint" ]
```

Comment:
For this commit ("Create a new container to run migrations") it would be good to explain in the commit message:

- What's the motivation? It says migrations now only run once per deploy but why is this a good thing, what were we experiencing before?
- The delay that we rely on that makes this approach work
- The worst-case scenario, where this delay isn't long enough and database preparation takes longer than it takes the worker and web containers to start up. Yesterday we were talking and you said it's not a problem because those containers would just keep failing with a pending migrations error and trying to restart. This is useful information for future devs.

Reply:
Yeah, that makes sense. Sorted 👍